cms/djangoapps/contentstore/tasks.py

```diff
@@ -58,7 +58,7 @@ def rerun_course(source_course_key_string, destination_course_key_string, user_i
             # it's possible there was an error even before the course module was created
             pass
 
-        return "exception: " + unicode(exc)
+        return ("exception: " + unicode(exc))[:999]  # don't exceed 1000 chars
```

Comment:
@nasthagiri hm, thanks for the catch - I think I'm doing the right thing but in the wrong place.  

Here's a gist of the trace: https://gist.github.com/jimabramson/fa4839466ec04d5b7786  ...which suggests the string truncation actually belongs in course_action_state/managers.py.